CommandLine/testing/Tests/room_transition_test.gmx/objects/obj_0.object.gmx

```diff
@@ -25,7 +25,20 @@
         <arguments>
           <argument>
             <kind>1</kind>
-            <string>room_goto(rm_1);</string>
+            <string>if (room == rm_0) {
+    gtest_assert_eq(draw_get_color(), c_white);
+    gtest_assert_eq_eps(draw_get_alpha(), 1.0);
+
+    draw_set_color(c_red);
+    draw_set_alpha(0.2);
+    room_goto(rm_1);
+} else if (room == rm_1) {
+    // ensure that room transition did NOT reset color/alpha
+    // GM does not reset ANY render state when changing rooms
+    gtest_assert_eq(draw_get_color(), c_red);
+    gtest_assert_eq_eps(draw_get_alpha(), 0.2);

```

Comment:
@JoshDreamland pay attention to this part when you review this. In case it's not obvious why I chose `0.2` it's because 51/255 = `0.2` in a calculator. When I tried `0.5` I got: `Failure: Not true that draw_get_alpha() (which is 0.498039) is approximately equal to 0.5 (which is 0.5).`.

Reply:
Use the method that takes an epsilon. It tests approximate equality.